pycqed/analysis/PSD/standard_arches_psd.py

```diff
@@ -189,9 +189,9 @@ def plot_coherence_times_freq(flux, freq, sensitivity,
                          figname='Coherence_times.PNG'):
     f, ax = plt.subplots()
 
-    ax.plot(freq/1e9, T1/1e-6, 'o', color='C3', label='$T_1$')
-    ax.plot(freq/1e9, Tramsey/1e-6, 'o', color='C2', label='$T_2^*$')
-    ax.plot(freq/1e9, Techo/1e-6, 'o', color='C0', label='$T_2$')
+    ax.plot(freq/1e9, T1/1e-6, 'o', color='b', label='$T_1$')
```

Comment:
I think these changes should be reverted . 
The C3/C1 uses whatever the corresponding colors are in the colormap (and should correspond to the pretty versions of rgb).